apps/services/user-profile/src/app/v2/test/user-profile.spec.ts

```diff
@@ -109,17 +114,16 @@ describe('UserProfileController', () => {
 
     it('GET /v2/user/.national-id should return 200 with the UserProfileDto when the User Profile exists in db', async () => {
       // Arrange
-      const fixtureFactory = new FixtureFactory(app)
-      const userProfile = fixtureFactory.createUserProfile(testUserProfile)
+      await new FixtureFactory(app).createUserProfile(testUserProfile)
```

Comment:
As we now have `beforeAll` the in memory db is shared for all tests so this profile will exist in other test cases. It would also make sense to make the `FixtureFactory` available in the `describe` scope like the `server`. 
I think we should use the `beforeEach` to clear the db before each test case so we prevent any data leakage between test cases.